time_tracker_api/projects/projects_model.py

```diff
@@ -21,13 +21,15 @@ def create_dao() -> ProjectDao:
     from time_tracker_api.database import COMMENTS_MAX_LENGTH
     from time_tracker_api.sql_repository import SQLCRUDDao, AuditedSQLModel
 
-    class ProjectSQLModel(db.Model, AuditedSQLModel):
+    class ProjectSQLModel(db.Model):
         __tablename__ = 'project'
-        id = db.Column(db.Integer, primary_key=True)
+        id = db.Column(db.String, primary_key=True)
```

Comment:
Whenever you see a `String` that is related to a `UUID`, you should probably set a max number of characters, like 64 maybe.

Reply:
Also SQL Alchemy has [multiple ways of generating UUID content](https://stackoverflow.com/questions/183042/how-can-i-use-uuids-in-sqlalchemy) and you need to do it for the attribute `id`.

Reply:
So that you are sure that such UUID works, you should probably change a bit the SQL Repository tests so that the model uses a UUID as primary key and verify that such id is actually being generated.